crowdin.yml

```diff
@@ -0,0 +1,8 @@
+project_id_env: REACT_APP_CROWDIN_PROJECT_ID # Your Crowdin project ID
+api_token_env: REACT_APP_CROWDIN_API_KEY
+
+preserve_hierarchy: true
+
+files: # Paths to the source and translation files
+  - source:  /translations/en/*
```

Comment:
I think we only need .po files here (By running `yarn` it will some ts files here as well.)

Reply:
I also tried to run: `yarn i18n:syncTranslations `


```
Downloading java runtime environment for version 11
✔️  Fetching project info     
✔️  Building ZIP archive with the latest translations
✔️  Building translation (100%)    
✔️  Downloading translations     
✔️  Extracting archive     
⚠️  Couldn't find any file to download
⚠️  Downloaded translations don't match the current project configuration. The translations for the following sources will be omitted (use --verbose to get the list of the omitted translations):
        - locales/en/messages.d.ts (3)
        - locales/en/messages.po (3)
        - locales/en/messages.ts (3)
Visit the https://crowdin.github.io/crowdin-cli/faq for more details

```

Reply:
It was fixed.